README.rst

```diff
@@ -14,6 +14,21 @@ since only changes are stored.
 The authenticated encryption technique makes it suitable for backups to not
 fully trusted targets.
 
+Quick Install
+~~~~~~~~~~~~~
+
+Open a terminal in the folder where you downloaded Borg Backup and run (without "$"):
+
+``$ chmod +x scripts/install``
+
+And after:
+
+``$./scripts/install``
+
+
+Probably ask you to enter your password, and it's done!
+
+
```

Comment:
no install docs in the readme please, move this to an appropriate section of the documentation.